conf/modules.config

```diff
@@ -250,5 +250,12 @@ if (params.aligner == 'cellrangermulti') {
                 mode: params.publish_dir_mode
             ]
         }
+        withName: BAMTOFASTQ10X {
+            publishDir = [
+                path: "${params.outdir}/${params.aligner}/bam2fastq",
+                mode: params.publish_dir_mode
+            ]
+            ext.args = "--reads-per-fastq=2200000000"
```

Comment:
The default for reads-per-fastq is 50.000.000, and I guess it won't make a difference whether the `.fastq`s are split into multiple files, but I was just following a guide very strictly.

Reply:
Cellranger is completely agnostic of how the fastqs are split, see also here: https://github.com/nf-core/modules/blob/master/modules/nf-core/cellranger/count/README.md, we can as well stay with the default.